cmd/otelcol/main.go

```diff
@@ -61,6 +62,10 @@ func main() {
 		checkRuntimeParams()
 	}
 
+	// Allow dumping configuration locally by default
+	// Used by support bundle script
+	os.Setenv(configServerEnabledEnvVar, "true")
```

Comment:
fyi: if the port is not available it would fail to start - I don't recall the exact message in this case.

Reply:
bind: port already in use -- already hit it :)